contrib/extractor_scripts/ExtractResources.sh

```diff
@@ -73,6 +74,7 @@ then
   USE_AD="1"
   USE_VMAPS="1"
   USE_MMAPS="1"
+  USE_MODELS="1"
```

Comment:
is this a significant extraction time? cos if not id just do it always like cameras. Also i assume ur extracting all of them and not just the ones needed for vehicles?

Reply:
it's not, takes a minute or so on my system
I am extracting all creature m2

I added it to be default when using extract all, I think that should be enough